misc/parity/reports/algod2indexer_full.yml

```diff
@@ -17,6 +17,16 @@ definitions:
         description:
         - INDEXER: '"Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig\n* or null if unknown"'
         - ALGOD: '"Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig"'
+      total-box-bytes:
```

Comment:
Differs from `go-algorand`'s definition. I've [posed a question](https://github.com/algorand/go-algorand/pull/4149/files?show-deleted-files=true&show-viewed-files=true&file-filters%5B%5D=#r953316978) regarding whether this should be the case.

Reply:
Resolving as an issue has been opened in go-algorand: https://github.com/algorand/go-algorand/issues/4478